pkg/geoipupdate/config.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	PreserveFileTimes bool
 	Verbose           bool
 	RetryFor          time.Duration
+	Parallelism       int
```

Comment:
Maybe a comment here mentioning that this won't affect the behavior of the library would make sense.